tests/test_process_graph_validation.py

```diff
@@ -68,52 +74,13 @@ def test_6_graph_actiniamodule(self):
         """Run the validation test
         """
         response = self.app.post(
+            self.prefix +
             '/validation',
             data=json.dumps(ACTINIA_PROCESS),
             content_type="application/json",
             headers=self.auth)
         self.assertEqual(response.status_code, 200)
 
-    def no_support_test_6_graph_zonal_statistics(self):
```

Comment:
This part has already been deleted from master ?

Reply:
I thought I pulled before I started :innocent: I fixed it.